showpass/Showpass-E2E-tests
# TA-21-Add a spec to verify that a new Billing Address can be properly auto-populated

**Description:**
When a user adds a new payment method, they are required to populate the new billing address form as well. This form can be auto-populated if the address, the user is searching for, found and selected. Each field within this ‘Billing Address’ form gets filled automatically.

Steps:
- Navigate to https://beta.showpass.com/account/profile/update-payment/
- Input a valid address into the ‘Search Address’ field
- Ensure that the other form’s fields get auto-populated
- Verify that auto-populated values are correct
- Click ‘Cancel’

**Jira ticket:**
https://showpass.atlassian.net/browse/TA-21

**How to test:**
Run this command and ensure all specs pass
npx cypress run --spec "cypress/e2e/user-settings/profile.cy.js"

**Test execution results:**
<img width="1072" alt="billingAddressAutoPopulated" src="https://github.com/showpass/Showpass-E2E-tests/assets/138615449/34def70e-5b8d-438b-b7b7-c6a5376ee133">
